### ISSUE TITLE
stackConfigDir does not allow for a variable

### ISSUE THREAD

**Issue by rdanno** [pulumi/pulumi#11960]
I'm in a situation where i need to symlink this path to the directory of my configs... i have many dirs.

Can you please implement the ability to specify a bash variable to the path of the config dir?

eg.
```
stackConfigDir: ${TEST}
```

https://github.com/pulumi/pulumi/blob/bde6f5e5aa634f45ac26d95e224a509c5a58e5bb/sdk/go/common/workspace/paths.go#L105

**Comment by Frassle** [comment]
It's not clear how an envvar helps here? Wouldn't a relative path and symlink be more appropriate?

**Comment by rdanno** [comment]
You're right there's not much difference between an env var and a symlink here.

Is there any way we can use a Pulumi variable? I'd like to derive the config path from something defined in the config/codebase
